### PR TITLE
added: hamburger for responsiveness and refractored

### PR DESCRIPTION
**Pull request by xii-vi**
### Refractored some code.

mobile (9)

### CODE REVIEW

File: javascript/navbar.js [hunk 1]
```diff
@@ -6,8 +6,54 @@ const navbarComponent = () => document.querySelector(".create-navbar").innerHTML
...
+    <a class="p-4 flex anchor-tag" href="/pages/image.html">Image</a>
+    <a class="p-4 flex anchor-tag" href="/pages/input.html">Input</a>
+    <a class="p-4 flex anchor-tag" href="/pages/list.html">List</a>
+    <a class="p-4 flex anchor-tag" href="/pages/modal.html">Modal</a>
+    <a class="p-4 flex anchor-tag" href="/pages/navigation.html">Navigation</a>
+    <a class="p-4 flex anchor-tag" href="/pages/rating.html">Rating</a>
+    <a class="p-4 flex anchor-tag" href="/pages/slider.html">Slider</a>
+    <a class="p-4 flex anchor-tag" href="/pages/text-utilities.html">Text Utilities</a>
+    <a class="p-4 flex anchor-tag" href="/pages/toast.html">Toast</a>
+</div>
```
Comment (haarsh24) [inline javascript/navbar.js]: add formatting

**Review by haarsh24 — APPROVED** [review]
Good to merge

File: pages/grid.html [hunk 2]
```diff
@@ -57,8 +57,8 @@
...
-                    style="width: 100%; height: 180px; border:0; transform: scale(1); overflow:hidden;"
+                    src="https://carbon.now.sh/embed?bg=rgba%2874%2C144%2C226%2C1%29&t=material&wt=sharp&l=htmlmixed&width=&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Fira+Code&fs=14px&lh=150%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522grid-2%2522%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522grid-item%2522%253EGrid%2520Item%25201%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522grid-item%2522%253EGrid%2520Item%25202%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522grid-item%2522%253EGrid%2520Item%25203%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+                    style="width: 100%; height: 200px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (bishnoimukesh) [inline pages/grid.html:61]: avoid inline styling

**Review by bishnoimukesh — COMMENTED** [review]
suggestion given rest good to merge

File: pages/grid.html (hunk 2 above)
Comment (xii-vi) [reply pages/grid.html:61]: Cannot find a better solution for iframe from carbon.sh, that is why I am using inline one.
Please let me know if you find a better solution.

**Pull request merged**